app/assets/javascripts/application.js

```diff
@@ -1,4 +1,4 @@
-//= require jquery
+//= require jquery3
```

Comment:
This is an interim setup. Next is removal of "jquery_ujs", after which we won't need the `jquery-rails` Gem and can switch to getting jQuery from npm.